mani_skill/envs/tasks/digital_twins/base_real_agent.py

```diff
@@ -0,0 +1,111 @@
+from typing import Optional, Union
+
+import cv2
+import numpy as np
+import torch
+
+from mani_skill.agents.base_agent import BaseAgent
+from mani_skill.envs.scene import ManiSkillScene
+
+
+# converting sim controller to real controller, only requires changing a few functions
+class RealController:
```

Comment:
before we merge this i will have some other members in our lab take a pass at real robot control interfaces and take note of anything else we might need to track usually.